apps/src/javalab/PreviewPaneHeader.jsx

```diff
@@ -31,12 +30,16 @@ export default function PreviewPaneHeader({
           leftJustified
         />
       </PaneSection>
-      <PaneSection style={styles.headerSectionCenter}>
+      <PaneSection className={'pane-header-section pane-header-section-center'}>
         {showPreviewTitle && (
           <PaneSection style={styles.headerTitle}>{i18n.preview()}</PaneSection>
         )}
       </PaneSection>
-      <PaneSection style={styles.headerSectionRight}>
+      {/* This overflowX styling should ideally be in style.scss */}
```

Comment:
Could you add to the comment why it's not in css? (edit: or maybe just "see style.scss for details")